proof-new/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains an example script to execute cvc5 to see if new rewrite
```

Comment:
Again, we shouldn't introduce a new folder in this PR.  I would remove this file. 

More specific documentation could be added about the DSL, although I forget where we decided this should live.

Reply:
delteed